plugins/lua/sort.lua

```diff
@@ -181,6 +205,173 @@ local function make_sort_by_skill_asc(sort_skill)
     end
 end
 
+-- Statistical rating that is bigger for more potent dwarves in long run melee military training
```

Comment:
could you add a comment linking back to the data (and algorithm) that you used to derive the weights? Maybe an online doc or spreadsheet?

Reply:
I have a reddit post explaining these coefficients with code and data linked there https://www.reddit.com/r/dwarffortress/comments/1617s11/enhancing_military_candidate_selection_part_2/. Should I add it to sort.rst or somewhere else?

Reply:
yeah, adding the link to the docs would be sufficient.